zerver/templatetags/app_filters.py

```diff
@@ -85,4 +85,5 @@ def render_markdown_path(markdown_file_path, context=None):
     template = loader.get_template(markdown_file_path)
     markdown_string = template.render(context)
     html = md_engine.convert(markdown_string)
-    return mark_safe(html)
+    html_template = engines['backends'].from_string(html)
```

Comment:
Can you add "NAME" field in Jinja2 template backend entry under `TEMPLATES` setting  in `settings.py`? That way you can get the Jinja2 template like this: `engines['Jinja2']`.